NPE in AutoForeground.promoteForeground() - App crashes after inputing password.

Logging to WordPress.com, the crash happens after you input correct password and "Log in" progress dialog is dismissed.

Only happens with 2fa enabled account.

```
E/AndroidRuntime: FATAL EXCEPTION: main
                  Process: org.wordpress.android, PID: 29104
                  java.lang.RuntimeException: Unable to unbind to service org.wordpress.android.login.LoginWpcomService@6ba73e8 with Intent { cmp=org.wordpress.android/.login.LoginWpcomService }: java.lang.NullPointerException: Attempt to invoke interface method 'boolean org.wordpress.android.util.AutoForeground$ServiceState.isInProgress()' on a null object reference
                      at android.app.ActivityThread.handleUnbindService(ActivityThread.java:3408)
                      at android.app.ActivityThread.-wrap27(Unknown Source:0)
                      at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1687)
                      at android.os.Handler.dispatchMessage(Handler.java:106)
                      at android.os.Looper.loop(Looper.java:164)
                      at android.app.ActivityThread.main(ActivityThread.java:6494)
                      at java.lang.reflect.Method.invoke(Native Method)
                      at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
                      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
                   Caused by: java.lang.NullPointerException: Attempt to invoke interface method 'boolean org.wordpress.android.util.AutoForeground$ServiceState.isInProgress()' on a null object reference
                      at org.wordpress.android.util.AutoForeground.promoteForeground(AutoForeground.java:145)
                      at org.wordpress.android.util.AutoForeground.onUnbind(AutoForeground.java:123)
                      at android.app.ActivityThread.handleUnbindService(ActivityThread.java:3394)
                      at android.app.ActivityThread.-wrap27(Unknown Source:0) 
                      at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1687) 
                      at android.os.Handler.dispatchMessage(Handler.java:106) 
                      at android.os.Looper.loop(Looper.java:164) 
                      at android.app.ActivityThread.main(ActivityThread.java:6494) 
                      at java.lang.reflect.Method.invoke(Native Method) 
                      at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438) 
                      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807) 
```

--- a/libs/login/WordPressLoginFlow/build.gradle
+++ b/libs/login/WordPressLoginFlow/build.gradle
@@ -31,7 +31,7 @@ android {
 }
 
 dependencies {
-    implementation ('org.wordpress:utils:1.20.0-beta6') {
+    implementation ('org.wordpress:utils:1.20.0-beta7') {
         exclude group: "com.mcxiaoke.volley"
     }
 

--- a/libs/utils/WordPressUtils/build.gradle
+++ b/libs/utils/WordPressUtils/build.gradle
@@ -33,7 +33,7 @@ android {
     buildToolsVersion '26.0.2'
 
     defaultConfig {
-        versionName "1.20.0-beta6"
+        versionName "1.20.0-beta7"
         minSdkVersion 15
         targetSdkVersion 25
     }

--- a/libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/AutoForeground.java
+++ b/libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/AutoForeground.java
@@ -120,7 +120,10 @@ public abstract class AutoForeground<StateClass extends ServiceState>
     @Override
     public boolean onUnbind(Intent intent) {
         if (!hasConnectedClients()) {
-            promoteForeground();
+            final StateClass state = getState();
+            if (state != null && state.isInProgress()) {
+                promoteForeground(state);
+            }
         }
 
         return true; // call onRebind() if new clients connect
@@ -140,12 +143,9 @@ public abstract class AutoForeground<StateClass extends ServiceState>
         return getEventBus().hasSubscriberForEvent(mStateClass);
     }
 
-    private void promoteForeground() {
-        final StateClass state = getState();
-        if (state.isInProgress()) {
-            startForeground(NOTIFICATION_ID_PROGRESS, getNotification(state));
-            mIsForeground = true;
-        }
+    private void promoteForeground(StateClass currentState) {
+        startForeground(NOTIFICATION_ID_PROGRESS, getNotification(currentState));
+        mIsForeground = true;
     }
 
     private void background() {

